show/interfaces/__init__.py

```diff
@@ -1,14 +1,32 @@
 import json
+import os
 
 import click
 from natsort import natsorted
 from tabulate import tabulate
+from collections import OrderedDict
 
+from sonic_py_common import device_info
+from swsssdk import ConfigDBConnector
 from sonic_py_common import multi_asic
+from portconfig import get_child_ports
 import utilities_common.cli as clicommon
 import utilities_common.multi_asic as multi_asic_util
 import portchannel
 
+PLATFORM_JSON = 'platform.json'
```

Comment:
@samaity : Hi Sangita, as I know, we do not provide platform.json like this. We need a call to device_info.get_path_to_port_config_file(). Kindly confirm, if yes, I will mark it as a change needed.

Reply:
@praveen-li, I saw device_info.get_path_to_port_config_file() logic, I think its better way to find PLATFORM.json, so PR was updated.
Please take a look.

Reply:
We didn't have `get_path_to_port_config_file` available when the show breakout functionality was added. This is absolutely the better way to get the path. Thanks for the update!